### ISSUE TITLE
Add VMAD Property fails

### ISSUE THREAD

**Issue by fireundubh** [matortheeternal/zedit#27]
When I try to add a property to a script, I receive the following error:

```
Error: Failed to create new element at: 13, "."
Access violation at address 08D1F842 in module 'XEditLib.dll'. Write of address 0A0DCF8F
```

Tested on record type: TERM

Version: 0.3.0

**Comment by fireundubh** [comment]
Update: The property is added, but the view is not refreshed. I had to manually reload the record (via drag-and-drop) to see the new properties.

**Comment by matortheeternal** [comment]
Yes, this is a known issue but I'm not yet certain what the fix should be.  It's something in xEdit code.

**Comment by xieve** [comment]
In case someone stumbles across this because the same error occurs with scripts, here's a workaround (written for the Automate Modal, but I'm sure it should be easy to adapt for patchers)

```JS
function AddArrayItemWorkaround(id, path, subpath, value) {
	try {
		xelib.AddArrayItem(id, path, subpath, value)
	} catch {}
	return xelib.GetElement(id, `${path}\\[0]`)
}
```
